homeassistant/components/transmission/config_flow.py

```diff
@@ -70,9 +67,6 @@ async def async_step_user(
                     and entry.data[CONF_PORT] == user_input[CONF_PORT]
                 ):
                     return self.async_abort(reason="already_configured")
-                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
-                    errors[CONF_NAME] = "name_exists"
```

Comment:
We can remove this error from the strings.json.

(Also for a follow-up PR, you can use `self._async_abort_entries_match()` instead of this for loop :)